for_attendees.md

```diff
@@ -1,120 +1,127 @@
-# Remote Learning Thoughts for Workshop Attendees
+# Remote Learning Instructions for Workshop Attendees
 
 ### Introduction
 
-This document is an attempt to list all many points that need to be considered when attending
- a class that offers a remote option or is taught entirely remotely.  As in-person support is
-  not available, it is imperative that proper planning and sufficient lead time are given to
-   ensure a smooth and enjoyable class experience. Additionally, we&#39;d like to encourage
-    as much interaction as possible and when reasonable.
+This document lists points to consider when attending a workshop that offers a remote option
+or is taught entirely remotely. Most importantly, **ensure that all required software is
+installed and working correctly ahead of time**. As in-person support is not available, it is
+imperative that proper planning and sufficient lead time are given to ensure a smooth and
+enjoyable class experience. Additionally, we&#39;d like to encourage as much interaction as
+possible and when reasonable.
 
 For the purposes of this document, we will highlight solutions in [Zoom](http://zoom.us/),
- though likely solutions exist in other video conferencing software.
-
-We recommend the following guides on remote learning for reading:
-
- * [https://carpentries.github.io/instructor-training/guide/index.html#zoom](https://carpentries.github.io/instructor-training/guide/index.html#zoom)    PLEASE READ
-[https://teachremotely.harvard.edu/best-practices](https://teachremotely.harvard.edu/best-practices)
- * [http://www.russpoldrack.org/2020/01/talking-remotely-lessons-learned-so-far.html](http://www.russpoldrack.org/2020/01/talking-remotely-lessons-learned-so-far.html)
-  * [https://github.com/jfiksel/github-classroom-for-teachers](https://github.com/jfiksel/github-classroom-for-teachers)
+though solutions exist in other video conferencing software. A number of useful Zoom guides
+are available at [https://support.zoom.us/](https://support.zoom.us/).
 
 ### Zoom / Video Conference Setup
 
-Many guides are available online since this mass transition to online work, teaching, and
- learning. We do recommend the ones above, but the following tips are highly recommended
+Many guides are available online, and we've summarized some of the most important tips here.
 
-1. Setup or verify your computer and learning location a day or two in advance, and test it
- out with a colleague. The day-of is not the best time to learn about problems.
+1. Setup or verify your computer and learning location a day or two in advance. Test your
+   software at [https://zoom.us/test](https://zoom.us/test) and with a colleague. The day-of
+   is not the best time to learn about problems!
 
```

Comment:
I think giving the test link makes it more likely that people will actually test.